roles/splunk/tasks/configure_firewall.yml

```diff
@@ -0,0 +1,33 @@
+---
+- name: Ensure {{ firewall_service }}  package is installed
+  ansible.builtin.package:
+    name: "{{ firewall_service }}"
+    state: present
```

Comment:
You need to have `become: True`, otherwise the install will fail. You need this for all tasks that require superuser privileges.

Reply:
Fixed with https://github.com/splunk/ansible-role-for-splunk/pull/157/commits/f93c8a18104136f20458ce214e3770068b9d1f80